ccmlib/scylla_node.py

```diff
@@ -950,6 +950,8 @@ def __update_yaml(self):
                 except KeyError:
                     data[name] = full_options[name]
 
+        if 'alternator_port' in data or 'alternator_https_port' in data:
```

Comment:
it could also be considered to have a property of is_alternator_enabled. or a list of all enabled features (but that may be out of ccm scope).

Reply:
you are right, this is not the scope of this PR, and not CCM, AFAIK.